src/SetupPage.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+
+export default class SetupPage extends React.Component {
+  render() {
+    return (
+      <div>
+        <SetupForm />
+        <WhatIsIt />
+      </div>
+    );
+  }
+}
+
+class SetupForm extends React.Component {
+  render() {
+    document.title = `What's deployed? (React)`;
+
+    return (
+      <form>
+        <div className="form-group">
+          <label>Owner</label>
+          <input
+            type="text"
+            name="owner"
+            className="form-control"
+            id="owner"
+            value="mozilla"
+          />
+        </div>
+        <div className="form-group">
+          <label>Repository</label>
+          <input
+            type="text"
+            name="repo"
+            className="form-control"
+            id="repo"
+            placeholder="e.g. airmozilla"
+          />
+        </div>
+        <div className="form-group revisions">
+          <label>Revision URLs</label>
+          <input
+            type="text"
+            name="name[]"
+            className="form-control name"
+            placeholder="Dev"
+          />
+          <input
+            type="text"
+            name="url[]"
+            className="form-control url"
+            placeholder="e.g. https://air-dev.allizom.org/media/revision or https://example.com/__version__"
+          />
+        </div>
+        <p>
+          <button type="button" className="btn btn-secondary more">
+            Add Row
+          </button>
+        </p>
+        <div>
+          <button className="btn btn-primary">
+            Generate <i>What's Deployed</i> Table
```

Comment:
Why is GitHub doing this?!

Reply:
Github doesn't really understand JSX, and so it thinks this single quote is the start of an unterminated string. Frustrating

Reply:
Yeah, but it's not the first time I've used `'` characters in GitHub PRs. Just got me worried that we're doing something wrong this time.

Reply:
Relax. [I'lll fix it, eventually.](https://github.com/Alhadis/language-js)  I started but something more important kinda stole amy attention for a little while...

The grammar I began work on would aim to be THE only ES grammar  you'd need, supporting ESX and Flow Static Type Hintings, and whatnot... basically, it could be the one JS grammar you could use *everywhere* and not have to forget about subtle discrepancies between Atom and VSCode's pattern-handling, of that of Sublime Text iand so forth...